addons/point_of_sale/tests/test_point_of_sale_flow.py

```diff
@@ -3,7 +3,7 @@
 
 import time
 from freezegun import freeze_time
-from datetime import datetime
+from datetime import datetime, timedelta
```

Comment:
```suggestion
from datetime import timedelta
```
Should solve the style check on runbot.